clicks.view.lkml

```diff
@@ -1,86 +1,118 @@
+# Details on the derived.clicks table is in the Snowplow Web Model documentation:
+# https://github.com/snowplow-proservices/ca.bc.gov-snowplow-pipeline/blob/master/jobs/webmodel/README.md#4-clicks
+
 view: clicks {
   sql_table_name: derived.clicks ;;
 
-
+  # duplicate
   dimension: p_key {
+    description: "The primary key, which is one of: User ID, Domain User ID, Session ID, or Click ID."
     primary_key: yes
     sql: ${user_id} || ${domain_userid} || ${session_id} || ${click_id} ;;
   }
 
   # DIMENSIONS
 
   # Application
+  # This section contains fields that are duplicated across over view files in this project
 
   dimension: app_id {
+    description: "The application idientifier."
     type: string
     sql: ${TABLE}.app_id ;;
     group_label: "Application"
   }
 
   # this is only available for WebTrends data
   dimension: dcs_id {
+    description: "The identifier generated by a hit, a Webtrends identifier."
     type: string
     sql: ${TABLE}.dcs_id ;;
     group_label: "Application"
   }
 
   # User
+  # This section contains fields that are duplicated across over view files in this project
 
   dimension: user_id {
+    description: "A unique identifier set by the business."
     type: string
     sql: ${TABLE}.user_id ;;
     group_label: "User"
   }
 
   dimension: domain_userid {
+    description: "The user identifier set by Snowplow using a first party cookie."
     type: string
     sql: ${TABLE}.domain_userid ;;
     group_label: "User"
   }
 
   dimension: network_userid {
+    description: "The user identifier set by Snowplow using a third party cookie."
     type: string
     sql: ${TABLE}.network_userid ;;
     group_label: "User"
     hidden: yes
   }
 
   # Session
+  # This section contains fields that are duplicated across over view files in this project
 
   dimension: session_id {
+    description: "A visit or session identifier."
     type: string
     sql: ${TABLE}.session_id ;;
     group_label: "Session"
   }
 
   dimension: session_index {
```

Comment:
isn't this a hidden field in the sessions view?

Reply:
No, in the sessions.view file this is not hidden.